r/tests/testthat/test-data-type.R

```diff
@@ -395,6 +395,18 @@ test_that("decimal type and validation", {
   expect_error(decimal(4, NA), '"scale" must be an integer')
 
   expect_r6_class(decimal(4, 2), "Decimal128Type")
+
+  expect_error(decimal128())
+  expect_error(decimal128("four"), '"precision" must be an integer')
+  expect_error(decimal128(4))
```

Comment:
Is this an error that the R package raises? If so, should we assert what it says?

Reply:
Same as for the other arg to `decimal128()`, this is a generic R error: `Error in decimal128(4) : argument "scale" is missing, with no default`. I merely duplicated the tests we had for `decimal()`.

Reply:
Done.